web/app/Services/Deepwell/DeepwellService.php

```diff
@@ -67,24 +67,28 @@ public function translate(string $locale, string $key, array $values = []): ?str
     }
 
     // User
-    public function getUserById(int $id): ?object
+    public function getUserById(int $id, ?string $detail = null): ?object
     {
-        $resp = $this->client->get("user/id/$id");
+        $query = $detail ? "?detail=$detail" : '';
+        $resp = $this->client->get("user/id/$id$query");
```

Comment:
Guzzle has a way of [passing in query parameters without manually building them](https://docs.guzzlephp.org/en/stable/quickstart.html#query-string-parameters). Also I don't see anything wrong with having `string $detail = 'identity'` and just passing the query parameter each time.

Reply:
lmao yeah tbh I was being lazy because the `options` field was untyped in my IDE (because PHP arrays gdi)